jellyfin-platform-android/build.gradle.kts

```diff
@@ -49,13 +49,20 @@ tasks.create<Jar>("sourcesArtifact") {
 	from(android.sourceSets["main"].java.srcDirs)
 }
 
+val javadocJar by tasks.creating(Jar::class) {
+	dependsOn(tasks.getByName("dokkaJavadoc"))
+	archiveClassifier.set("javadoc")
+	from("$buildDir/dokka/javadoc")
+}
+
 // Because of limitations in the android plugin
 // the publishing definition should be inside the "afterEvaluate" block
 afterEvaluate {
 	publishing.publications.create<MavenPublication>("default") {
 		from(components["release"])
 
 		artifact(tasks["sourcesArtifact"])
+		artifact(tasks["javadocJar"])
```

Comment:
`javadocJar` reference won't work here?

Reply:
Nope because of the `afterEvaluate` block, Android is special